durablefunctionsmonitor.dotnetisolated.core/Common/Auth.cs

```diff
@@ -334,6 +324,48 @@ private static string TryGetHubNameFromHostJson()
             }
         }
 
+        private static async Task<ClaimsPrincipal> GetClaimsPrincipal(HttpRequestData request, string userNameClaimName, string rolesClaimName)
+        {
+            // First trying the request object
+            var principal = new ClaimsPrincipal(request.Identities);
+
+            if (principal.Identity != null && principal.Identity.IsAuthenticated && principal.HasClaim(c => c.Type == userNameClaimName))
+            {
+                return principal;
+            }
```

Comment:
If I understand correctly, this is the case where the user is already authenticated, so we can return immediately. To help me build context, I'd appreciate if we could add this comment:
```suggestion
            // authentication has already established, so we can return immediately
            if (principal.Identity != null && principal.Identity.IsAuthenticated && principal.HasClaim(c => c.Type == userNameClaimName))
            {
                return principal;
            }
```

Reply:
OK.